source/ed/ed_reader.cpp

```diff
@@ -971,7 +971,11 @@ void EdReader::fill_stop_times(nt::Data& data, pqxx::work& work) {
 
         const_it["arrival_time"].to(stop.arrival_time);
         const_it["departure_time"].to(stop.departure_time);
-        const_it["local_traffic_zone"].to(stop.local_traffic_zone);
+        if (const_it["local_traffic_zone"].is_null()){
```

Comment:
I think it's best to set it only if not null, that way the default value is defined only in one place